cmd/crictl/container.go

```diff
@@ -624,15 +624,11 @@ func ListContainers(client pb.RuntimeServiceClient, opts listOptions) error {
 		createdAt := time.Unix(0, c.CreatedAt)
 		ctm := units.HumanDuration(time.Now().UTC().Sub(createdAt)) + " ago"
 		if !opts.verbose {
-			id := c.Id
-			if !opts.noTrunc {
-				id = strings.TrimPrefix(c.Id, "")
-				if len(id) > truncatedIDLen {
-					id = id[:truncatedIDLen]
-				}
-			}
+			id := getTruncatedID(opts.noTrunc, c.Id, "")
+			// Now c.Image.Image is imageID in kubelet.
+			imageID := getTruncatedID(opts.noTrunc, c.Image.Image, "sha256:")
```

Comment:
Can we use `go-digest` to parse the id here? We should only truncate when it is an image id.

The kubelet implementation may change in the future, we should not rely on it.

Reply:
done